hapi/hedera-protobufs/platform/tss/tss_vote_map_key.proto

```diff
@@ -10,11 +10,9 @@
  */
 syntax = "proto3";
 
-package com.hedera.hapi.node.state.tss;
+package com.hedera.hapi.platform.tss;
```

Comment:
Same as above - shouldn't we delete this file? Or move the models to com.hedera.hapi.services.auxiliary.tss ?

Reply:
Actually, these appear to be moved from `state/tss`.  If they're intended as keys for the virtual map for TSS values in state, they should probably remain in `state/tss`.

Reply:
Done (for both files)